packages/sources/dxdao/src/endpoint/tvl.ts

```diff
@@ -0,0 +1,65 @@
+import { Requester, Validator, Logger } from '@chainlink/ea-bootstrap'
+import { Config, ExecuteWithConfig } from '@chainlink/types'
+import { getRpcLatestAnswer } from "@chainlink/ea-reference-data-reader"
+import { JSON_RPC_URL } from '../config'
+import { ethers, BigNumber } from 'ethers'
+
+export const NAME = 'TVL'
+
+const customParams = {
+  wethContractAddress: true,
+  pairContractAddress: true,
+  xdaiEthUsdPriceFeedAddress: true,
```

Comment:
Also thought about naming this something more generic such as `priceFeedModifierAddressOnXDai` though I'm not sure why we would use a feed other than ETH-USD.  Also interested in hearing any thoughts.